adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/model/exceptions/AdapterException.java

```diff
@@ -0,0 +1,10 @@
+package org.jvalue.ods.adapterservice.adapter.model.exceptions;
+
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
```

Comment:
I would have assumed that lombok has some help for exceptions, but seems like the PR is not yet merged: https://github.com/rzwitserloot/lombok/pull/2702